config/initializers/active_metrics.rb

```diff
@@ -0,0 +1 @@
+ActiveMetrics::Base.establish_connection(:adapter => "miq_postgres", :database => "metrics_spike")
```

Comment:
Oh I guess it doesn't *really* matter, but if we switch adapters, I'm thinking the database name should be consistent.